boards/arm/thingy53_nrf5340/Kconfig

```diff
@@ -29,6 +29,10 @@ config BOARD_ENABLE_DCDC_HV
 	select SOC_DCDC_NRF53X_HV
 	default y
 
+config BOARD_ENABLE_USB_CDC
```

Comment:
I see it is used by adafruit_itsybitsy_nrf52840, but just CDC is not correct, we should introduce
`config BOARD_SERIAL_BACKEND_CDC_ACM` and use it consistently.